modin/core/execution/ray/common/deferred_execution.py

```diff
@@ -155,9 +156,9 @@ def exec(
             and self.flat_kwargs
             and self.num_returns == 1
         ):
-            result, length, width, ip = remote_exec_func.remote(
-                self.func, self.data, *self.args, **self.kwargs
-            )
+            result, length, width, ip = remote_exec_func.options(
+                resources=RayTaskCustomResources.get()
```

Comment:
I wonder if we should just call it `RayTaskResources`? (the same for `RayInitResources`) Since this config is used to pass values to `resources`.

Reply:
I would prefer to be explicit here as Ray itself calls it as custom resources - https://docs.ray.io/en/latest/ray-core/scheduling/resources.html#custom-resources.